server/config/requestHandler.js

```diff
@@ -49,4 +49,18 @@ module.exports = {
     res.send(responseJSON);
   },
 
-};
\ No newline at end of file
+  getPhotos: function(req, res) {
+    var responseJSON = {
+      photos: Photos
+    }
+
+    res.send(responseJSON);
+  },
+
+  postPhotos: function(req, res) {
+    var photo = req.body
+    Photos.push(photo);
```

Comment:
Makes sense. I deleted the photos array. We can play around with associating photos with users tomorrow.
btw -- Should Users be an array of user objects or should they be strings?